apps/src/gamelab/GameLabVisualizationColumn.jsx

```diff
@@ -79,6 +84,10 @@ class GameLabVisualizationColumn extends React.Component {
     mouseY: -1
   };
 
+  componentDidMount() {
+    this.setDanceLabSong();
+  }
```

Comment:
we shouldn't be waiting until the component mounts to set the song -- instead, you could dispatch an action to set the initial value of the `selectedSong` to the first value in the song library prior to rendering this component, which will then control the contents of the song selector. or, since the library is linked statically, you could just use the first song when setting the initial state in redux.

Reply:
I removed this componentDidMount and manually set an initial value for the redux state. This is a temporary solution - before the feature is released there will be an option in levelbuilder to set the initial song.